cli/replicaset/configsource.go

```diff
@@ -214,8 +214,8 @@ func (c *CConfigSource) Expel(ctx ExpelCtx) error {
 }
 
 // ChangeRole patches a config to add role to a config.
-func (c *CConfigSource) ChangeRole(ctx RolesChangeCtx, changeRoleFunc ChangeRoleFunc) error {
-	return c.patchConfigWithRoles(ctx, getCConfigRolesPath, changeRoleFunc, patchCConfigEditRole)
+func (c *CConfigSource) ChangeRole(ctx RolesChangeCtx, action RolesChangerAction) error {
```

Comment:
Comment for this function might also need an update:
`ChangeRole patches a config to add/remove role.`

Reply:
Yes, corrected, thank you!